src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-line-large"
+          data-qa="big"
+          action="/search"
+          method="get">
+      <img class="img-search-line-large"
+        src="./images/Search.svg"
+        alt="image">
+      <input class="text-search-line-large"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;">
+    </form>
```

Comment:
```suggestion
    <form 
      class="search-line-large"
      data-qa="big"
      action="/search"
      method="get"
    >
      <img 
        class="img-search-line-large"
        src="./images/Search.svg"
        alt="image"
      >
      <input 
        class="text-search-line-large"
        type="text"
        data-qa="keypress"
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```
Consider fixing indents according to the code style guide